### DISCUSSION TITLE
Windows compatability

### DISCUSSION THREAD

**Discussion by RocketManiac47** [maxileith/homebridge-appletv-enhanced#56] [Q&A] [answered]
Hi to the community,

I am currently running Homebridge on a Windows PC and not planning to change that. Is there a way to get the plugin working on Windows? I am running into issues when trying to install the plugin:

```logs
[#################.] - reify:semver: timing reifyNode:node_modules/homebridge-appletv-enh 
> @sebbo2002/node-pyatv@7.0.3 postinstall
> ./check.sh

'.' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code 1
npm ERR! path C:\Users\<User>\AppData\Roaming\npm\node_modules\homebridge-appletv-enhanced\node_modules\@sebbo2002\node-pyatv
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c ./check.sh
```

Is it planned to support Windows in the future?

I would like to here everybody's thoughts on that :)

**Comment by maxileith** [comment, marked as answer]
Hi @RocketManiac47,

thank you for your question. As stated in the README, I do not support Windows based homebridges, the error that you are mentioning was also mentioned in #4.

Windows is so different from Linux and Unix that it is just not feasible to make the plugin compatible with Windows. Especially considering the small userbase of Windows homebridges.

However, if you want to contribute to the project, feel free to do so. The error that you are experiencing is thrown by a postinstall script of a dependency called [node-pyatv](https://github.com/sebbo2002/node-pyatv). So this would be a good place to start.

Many Greetings
Maxi

**Reply by RocketManiac47** [reply]
Okay, that makes sense. Maybe I should consider switching to a Raspberry Pi or something similar. Have a nice evening :)
